backend/app/services/source_network.py

```
@@ -195,7 +213,15 @@ async def lookup_source_network(
             checked_at=checked_at,
             error=f"ASN lookup failed: {type(exc).__name__}.",
         )
-    return _from_cymru_txt(ip, records)
+    result = _from_cymru_txt(ip, records)
+    if result.asn and not result.as_name:
+        query = _asn_name_query(result.asn)
+        if query:
+            try:
+                result.as_name = _as_name_from_cymru_txt(await provider.lookup_txt(query))
+            except Exception:  # pylint: disable=broad-exception-caught
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)